src/app/builder/builder.component.html

```diff
@@ -64,6 +64,10 @@ <h3 *ngIf="feedUrl && !props">Loading episodes...</h3>
               <option value="all">All episodes in playlist</option>
             </select>
           </fieldset>
+          <fieldset *ngIf="playPlaylist">
+            <p>Optionally filter to a single itunes:season of your podcast. Leave blank to include all episodes.</p>
+            <input type="number" [(ngModel)]="props.playlistSeason" placeholder="Season Number" min="1" max="999">
```

Comment:
Is there any way to "unset" this once it is set? Not sure I am confident I know exactly how number inputs work, but worried that you could not clear this (not sure it matters)

Reply:
Good point.  In Chrome you can hit backspace ... but even that's not obvious.  I'll add some sort of "clear" button.

Reply:
Ended up switching this to a checkbox.  Just like all the other things!  Which also solves another problem by forcing the num-episodes to "all" when you select "Play a single season".

![image](https://user-images.githubusercontent.com/1410587/81311379-8f060880-9042-11ea-8de9-ed6a2f616719.png)